test/nodeMock/app.js

```diff
@@ -2,15 +2,22 @@
 
 const express = require('express');
 var bodyParser = require('body-parser');
+const minimist = require('minimist');
+
 
 let { requestMapping,configurations} = require('./reqResMapping');
 const { browser } = require('protractor');
-const CCDCaseConfig = require('./ccd/ccdCaseConfig/caseCreateConfigGenerator')
+const CCDCaseConfig = require('./ccd/ccdCaseConfig/caseCreateConfigGenerator');
+const CCDCaseDetails = require('./ccd/ccdCaseConfig/caseDetailsConfigGenerator');
+
+const { getDLCaseConfig} = require('../ngIntegration/mockData/ccdCaseMock');
+
 const port = 3001;
 
 
 class MockApp{
     init(){
+        this.intercepts =[];
```

Comment:
Being _really_ pedantic here, but spacing... recommend `this.intercepts = []`

Reply:
We need to make these changes, but not it this build - all changes from previous branch